app/serializers/spree_signifyd/user_serializer.rb

```diff
@@ -40,6 +40,10 @@ def aggregateOrderDollars
       completed_orders.sum(:total).to_f
     end
 
+    def phone
+      object.orders.order("created_at DESC").first.try!(:ship_address).try!(:phone)
```

Comment:
It would be, in the order serializer. Only since this is a User serializer it has no concept of a "current order". If this were something else, like a UserAccount serialize, it would probably be initialized with a current order and we'd just look there. This is why serialize OUR models was a bad idea.